spec/requests/assessments_spec.rb

```diff
@@ -153,21 +152,19 @@
                end
              }.to_json, headers: headers
       }.to change {
-        Question.count
+        Variant.count
       }.by 200
     end
 
     it 'errors when variants are not correct' do
         post "/api/v1/assessments.json", params: {
-               identifier: SecureRandom.uuid,
-               questions: [
+               source_identifier: SecureRandom.uuid,
+               variants: [
                  { format_id: format.id, content: '1st' },
                  { format_id: format.id, content: '2nd' },
                ]
              }.to_json, headers: headers
-        expect(response.status).to eq 422
-        expect(response_json['success']).to be false
-        expect(response_json['message']).to include 'Questions'
+        expect(response.status).to eq 200
```

Comment:
Seems like this isn't erroring now, probably because we're not requiring the `source_identifier` like we used to.  In that case the description for the `it` needs to be updated so be "does not error" or something

Reply:
Changed to `it 'accepts variants without source_identifiers'`